python/auxiliary.py

```diff
@@ -5,6 +5,8 @@
 
 from configurations.analysis_soepy_config import RESOURCES_DIR
 
+num_periods = 30
```

Comment:
Please use a capitalized name for the module-wide variable. `NUM_PERIODS`. Also in the other file.